### PR TITLE
add email functions in signup page

### PR DESCRIPTION
**Pull request by mohanagy**

### CODE REVIEW

File: backend/handlers/signuppost.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
-
+const mg = require('../mailer.js');
```
Comment (saralk) [inline backend/handlers/signuppost.js]: looks like you're missing this file

Comment (mohanagy) [inline backend/handlers/signuppost.js]: i wasn't know that patch ignore the new files

Comment (mohanagy) [inline backend/handlers/signuppost.js]: i fixed

File: backend/handlers/signuppost.js [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+
   user.createuser(req.payload, (err, result) => {
 
     if (err) {
       res({
-        msg: 'User already regestered'
+        msg: 'User already registered'
       }).code(400)
     } else {
+      mg.sendText('moha.buy@gmail.com', ['Recipient 1 <moha.buy@gmail.com>', 'moha.buy@gmail.com'],
```
Comment (saralk) [inline backend/handlers/signuppost.js]: let's make a wrapper function for this

**Comment by saralk** [comment]
and fix the tests

File: backend/utils.js [hunk 3]
```diff
@@ -9,4 +10,18 @@ const hash = (pass, cb) => {
...
+  sender, {},
+  function(err) {
+    if (err)
+      throw err ;
+  });
+}
+
+module.exports = {
+  hash: hash,
+  sendemail
```
Comment (saralk) [inline backend/utils.js]: wtf

**Pull request merged**